agent/bench-scripts/pbench-cyclictest

```diff
@@ -141,15 +141,15 @@ printf "\n" >>$benchmark_summary_txt_file
 printf "<pre>\n# these results generated with:\n# uperf %s\n\n" "$scrtip_name $orig_cmd" >$benchmark_summary_html_file
 printf "\n" >>$benchmark_summary_html_file
 
-pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir --sysinfo=$sysinfo beg
-
 # Start the tool meisters on each registered local/remote host
 pbench-tool-meister-start "${tool_group}"
 if [[ ${?} != 0 ]]; then
 	error_log "[${script_name}]: failed to start the tool meisters."
 	exit 1
 fi
 
+pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir --sysinfo=$sysinfo beg
```

Comment:
Now that Tool Meister is gathering sysinfo ... is there any reason we wouldn't always want to gather sysinfo immediately after starting and immediately before stopping each Tool Meister client? That is... could we remove the use of the special command and simply do it implicitly in the Tool Meister workflow?

Reply:
Yes, that is definitely a possibility.

We could move the default set of `sysinfo` collections performed into the `pbench-agent.cfg` file, and then when `pbench-tool-meister-start` begins, it could load those values from the global configuration.  The command line options for `--sysinfo` could be passed as an argument to `pbench-tool-meister-start` and then submitted with the base Tool Meister information posted in Redis for each instance.

Then `pbench-tool-meister-start` could initiate the client message instead of having each benchmark handle this flow.  This seems appealing to do in this commit.  Let me first get end-to-end collection of the system information first, and then I'll see about collapsing as described.

Reply:
Filed issue #1998 to track this work.